ArduPlane/quadplane.cpp

```diff
@@ -2691,7 +2709,7 @@ void QuadPlane::vtol_position_controller(void)
                                                                target_yaw_deg*100, true);
         } else {
             attitude_control->input_euler_angle_roll_pitch_euler_rate_yaw(plane.nav_roll_cd,
-                                                                          plane.nav_pitch_cd,
+                                                                          MAX(plane.nav_pitch_cd, q_fwd_nav_pitch_lim_cd),
```

Comment:
Same as Qloiter, should be up on 2694 so the `set_VTOL_roll_pitch_limit` has visibility.

Reply:
I've reworked it to move the angle limiting and forward thrust calculation to just after we get the demanded pitch angle from the position controller so that a consistent pitch angle demand is guaranteed for all downstream processing.